modules/ROOT/pages/viewing-api-analytics.adoc

```diff
@@ -146,13 +146,13 @@ The custom chart appears.
 * Select Application to display each application in a different column.
 * Select Violated Policy Name as the stack within each column.
 +
-image::create-chart-2.png[]
+image::create-chart-2.png["Stack chart configuration showing application grouping and violated policy name selection"]
```

Comment:
policy violation name?

Reply:
No, it's a selection in the UI: Violated Policy Name